pkgs/top-level/aliases.nix

```diff
@@ -558,6 +558,9 @@ mapAliases ({
   '';
   linux_latest_hardened = linuxPackages_latest_hardened;
 
+  # Added 2023-11-15 - Convert to `throw` after 23.11 release
+  linuxPackages_testing_bcachefs = linuxPackages_testing;
```

Comment:
This is still 6.7-rc1.

Reply:
If you're referencing the test I did earlier, I removed the aliases and all-packages changes to test it.